src/pybreaker.py

```diff
@@ -214,6 +230,98 @@ def remove_listener(self, listener):
             self._listeners.remove(listener)
 
 
+class CircuitBreakerStorage(object):
```

Comment:
@phillbaker Don't forget the docstrings when you finished implementing these new classes.